.github/workflows/publish.yml

```diff
@@ -22,14 +21,36 @@ jobs:
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
 
-      - run: npm ci
+      # Bump the version to the current
+      - name: 'Setting version of libraries to ${{ github.ref_name }} for libraries located in libraries/'
+        run: npm version ${{ github.ref_name }} -w libraries/ --force --no-git-tag-version
 
-      - run: npm run build
+      - name: 'Rebuilding affected libraries'
+        run: npm run build -w libraries/
 
-      - run: npm run publish-lib:dry
+      - name: 'Commit changes to master branch'
+        run: |
+          git config user.name "Six Webcomponents Release Bot"
+          git config user.email "<>"
+          git add -A
+          git commit -m "Automated lib version bump to ${{ github.ref }}"
+          git push origin master
+
+      - name: 'Publish all libraries to NPM'
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_PUBLISH_TOKEN }}
+        run: |
+          npm publish --dry-run --access=public libraries/ui-library/
+          npm publish --dry-run --access=public libraries/ui-library-react/
+          npm publish --dry-run --access=public libraries/ui-library-angular/dist/ui-library-angular/
 
-      - run: npm run publish-lib
+      - name: 'Create a release'
+        uses: actions/create-release@latest
         env:
-          NODE_AUTH_TOKEN: ${{ secrets.NPM_PUBLISH_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Is `GITHUB_TOKEN` already available? If not, who can create it?

Reply:
I believe anyone with admin access to the repo should be able to create it